plugins/providers/virtualbox/model/storage_controller_array.rb

```diff
@@ -4,9 +4,17 @@ module Model
       # A collection of storage controllers. Includes finder methods to look
       # up a storage controller by given attributes.
       class StorageControllerArray < Array
+        SATA_TYPE = "SATA".freeze
+        IDE_TYPE = "IDE".freeze
+        SUPPORTED_TYPES = [SATA_TYPE, IDE_TYPE].freeze
+
+        # TODO: hook into ValidateDiskExt capability
+        DEFAULT_DISK_EXT = [".vdi", ".vmdk", ".vhd"].map(&:freeze).freeze
```

Comment:
You can just grab the values from the constant directly:

```ruby
DEFAULT_DISK_EXT = Cap::ValidateDiskExt::DEFAULT_DISK_EXT
```

Reply:
It would also be good to merge this soon: https://github.com/hashicorp/vagrant/pull/11541
I've renamed that to more accurately reflect its value. There's `DEFAULT_DISK_EXT_LIST` and `DEFAULT_DISK_EXT`.

Reply:
Cool! I just hooked into `VagrantPlugins::ProviderVirtualBox::Cap::ValidateDiskExt.validate_disk_ext` so it should not be affected by the rename.